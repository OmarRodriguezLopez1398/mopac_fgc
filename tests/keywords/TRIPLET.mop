UHF MS=1.0
comment1
comment2
Ni 0.0 0.0 0.0

UHF TRIPLET
comment1
comment2
Ni 0.0 0.0 0.0

RHF MS=1.0 MECI OPEN(2,2)
comment1
comment2
Ni 0.0 0.0 0.0

RHF TRIPLET MECI OPEN(2,2)
comment1
comment2
Ni 0.0 0.0 0.0
