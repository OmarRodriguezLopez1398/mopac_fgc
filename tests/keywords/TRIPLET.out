 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:55 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  1.0
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
UHF MS=1.0
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Ni          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    6
                           NO. OF BETA  ELECTRONS =    4

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 UHF MS=1.0
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:55 2021




          FINAL HEAT OF FORMATION =         93.14412 KCAL/MOL =     389.71498 KJ/MOL


          COSMO AREA              =         45.84 SQUARE ANGSTROMS
          COSMO VOLUME            =         29.19 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =          7.028536 EV
          ALPHA SOMO LUMO (EV)    =         -7.666 25.286
          BETA  SOMO LUMO (EV)    =         -7.029 -1.611
          NO. OF ALPHA ELECTRONS  =          6
          NO. OF BETA  ELECTRONS  =          4
          MOLECULAR WEIGHT        =         58.7100         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.008 SECONDS
          COMPUTATION TIME        =          0.010 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Ni        0.000000000     0.000000000     0.000000000


           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
  -8.78851  -8.69801  -8.30726  -8.01253  -8.01253  -7.66583  25.28565  25.28565
  25.82271


           BETA EIGENVALUES 
  -7.61800  -7.17590  -7.17590  -7.02854  -1.61115  -0.54440  27.34732  27.67461
  27.67461


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Ni           0.000000       10.0000     1.00000     0.00000     9.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Ni   1.00000   0.00000   0.00000   0.00000   2.00000   2.00000   2.00000   2.00000   1.00000


                    (SZ)    =    1.000000
                    (S**2)  =    2.000000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   x^2-y^2  xz     z^2    yz     xy
         Atomic
    1 Ni  2.0000  1.000  0.000  0.000 -0.000  0.000  0.000 -0.000 -0.000  1.000
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:55 2021  *
 *  TRIPLET    - SPIN STATE DEFINED AS A TRIPLET
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
UHF TRIPLET
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Ni          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


 TRIPLET STATE CALCULATION


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    6
                           NO. OF BETA  ELECTRONS =    4

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 UHF TRIPLET
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:55 2021




          FINAL HEAT OF FORMATION =         93.14412 KCAL/MOL =     389.71498 KJ/MOL


          COSMO AREA              =         45.84 SQUARE ANGSTROMS
          COSMO VOLUME            =         29.19 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =          7.028536 EV
          ALPHA SOMO LUMO (EV)    =         -7.666 25.286
          BETA  SOMO LUMO (EV)    =         -7.029 -1.611
          NO. OF ALPHA ELECTRONS  =          6
          NO. OF BETA  ELECTRONS  =          4
          MOLECULAR WEIGHT        =         58.7100         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.020 SECONDS
          COMPUTATION TIME        =          0.018 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Ni        0.000000000     0.000000000     0.000000000


           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
  -8.78851  -8.69801  -8.30726  -8.01253  -8.01253  -7.66583  25.28565  25.28565
  25.82271


           BETA EIGENVALUES 
  -7.61800  -7.17590  -7.17590  -7.02854  -1.61115  -0.54440  27.34732  27.67461
  27.67461


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Ni           0.000000       10.0000     1.00000     0.00000     9.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Ni   1.00000   0.00000   0.00000   0.00000   2.00000   2.00000   2.00000   2.00000   1.00000


                    (SZ)    =    1.000000
                    (S**2)  =    2.000000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   x^2-y^2  xz     z^2    yz     xy
         Atomic
    1 Ni  2.0000  1.000  0.000  0.000 -0.000  0.000  0.000 -0.000 -0.000  1.000
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:55 2021  *
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 2 ELECTRONS IN 2 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  1.0
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF MS=1.0 MECI OPEN(2,2)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Ni          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  

      THERE ARE    4 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 4


                       NO. OF SINGLY OCCUPIED LEVELS =    2

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF MS=1.0 MECI OPEN(2,2)
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:55 2021




          FINAL HEAT OF FORMATION =         94.06259 KCAL/MOL =     393.55788 KJ/MOL


          COSMO AREA              =         45.84 SQUARE ANGSTROMS
          COSMO VOLUME            =         29.19 CUBIC ANGSTROMS

          NO. OF FILLED LEVELS    =          4
          AND NO. OF OPEN LEVELS  =          2
          MOLECULAR WEIGHT        =         58.7100         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.027 SECONDS
          COMPUTATION TIME        =          0.027 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Ni        0.000000000     0.000000000     0.000000000


           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
  -7.96227  -7.83386  -7.83386  -7.73259  -5.00928  -4.62084  26.42378  26.42378
  26.54582


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Ni           0.000000       10.0000     1.09502     0.00000     8.90498
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Ni   1.09502   0.00000   0.00000   0.00000   2.00000   2.00000   1.90498   2.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  1.0


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         5   6
          ENERGY

              1        1   1
         -3.6468       0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+   -3.646772    0.000000     1+  TRIPLET     ????

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:55 2021  *
 *  TRIPLET    - SPIN STATE DEFINED AS A TRIPLET
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 2 ELECTRONS IN 2 LEVELS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF TRIPLET MECI OPEN(2,2)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Ni          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


 TRIPLET STATE CALCULATION

      THERE ARE    4 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 4


                       NO. OF SINGLY OCCUPIED LEVELS =    2

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF TRIPLET MECI OPEN(2,2)
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:55 2021




          FINAL HEAT OF FORMATION =         94.06259 KCAL/MOL =     393.55788 KJ/MOL


          COSMO AREA              =         45.84 SQUARE ANGSTROMS
          COSMO VOLUME            =         29.19 CUBIC ANGSTROMS

          NO. OF FILLED LEVELS    =          4
          AND NO. OF OPEN LEVELS  =          2
          MOLECULAR WEIGHT        =         58.7100         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.035 SECONDS
          COMPUTATION TIME        =          0.036 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Ni          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Ni        0.000000000     0.000000000     0.000000000


           Empirical Formula: Ni  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
  -7.96227  -7.83386  -7.83386  -7.73259  -5.00928  -4.62084  26.42378  26.42378
  26.54582


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Ni           0.000000       10.0000     1.09502     0.00000     8.90498
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000    -0.000    -0.000     0.000
 SUM           -0.000    -0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Ni   1.09502   0.00000   0.00000   0.00000   2.00000   2.00000   1.90498   2.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  1.0


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         5   6
          ENERGY

              1        1   1
         -3.6468       0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+   -3.646772    0.000000     1+  TRIPLET     ????

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.10 SECONDS

 == MOPAC DONE ==
